udata/rdf.py

```diff
@@ -127,6 +129,20 @@
 HVD_LEGISLATION = "http://data.europa.eu/eli/reg_impl/2023/138/oj"
 TAG_TO_EU_HVD_CATEGORIES = {slugify_tag(EU_HVD_CATEGORIES[uri]): uri for uri in EU_HVD_CATEGORIES}
 
+AGENT_ROLE_TO_RDF_PREDICATE = {
+    "contact": DCAT.contactPoint,
+    "publisher": DCT.publisher,
+    "creator": DCT.creator,
+    "contributor": DCT.contributor,
```

Comment:
There are some contributor left in code I think

Reply:
Ah yes, sorry about that ;)